mitmproxy/web/app.py

```diff
@@ -292,6 +292,20 @@ def post(self, flow_id):
 
 class FlowContent(RequestHandler):
 
+    def post (self, flow_id, message):
+        # handle request later now just change response content
+
+        flow = self.flow
+        flow.backup()
+        content = self.request.files.values()[0][0].body
+        if (message == "response"):
```

Comment:
We can use `message = getattr(self.flow, message)` and then do the rest in a generic fashion.